Formula/terminator.rb

```diff
@@ -0,0 +1,46 @@
+class Terminator < Formula
+  include Language::Python::Virtualenv
+
+  desc "Multiple GNOME terminals in one window"
+  homepage "https://gnome-terminator.org"
+  url "https://github.com/gnome-terminator/terminator/archive/refs/tags/v2.1.1.tar.gz"
+  sha256 "ee1907bc9bfe03244f6d8074b214ef1638a964b38e21ca2ad4cca993d0c1d31e"
+  license "GPL-2.0-only"
+
+  depends_on "dbus"
+  depends_on "pygobject3"
+  depends_on "python@3.9"
```

Comment:
```suggestion
  depends_on "python@3.9"
  depends_on "six"
```
This can replace the `six` resource.